beacon-chain/core/validators/committees.go

```diff
@@ -139,3 +149,206 @@ func AttestationParticipants(
 	}
 	return participantIndices, nil
 }
+
+// CurrCommitteesCountPerSlot returns the number of crosslink committees per slot
```

Comment:
Can you choose a better name? 
This seems to imply that I am getting a map of `slot -> committeeCount`, but it returns sum of all committees?

Reply:
Discussed offline that this is to represent the number of committees for each shard.

Example: If this method returns 5, then it means there are 5 committees in each shard.

Suggestion was to add a clarifying example to the godoc